src/parse.rs

```diff
@@ -72,6 +72,8 @@ static FILE_EXTS: phf::Map<&'static str, &'static str> = phf_map! {
     "h" => "c",
     "tf" => "hcl",
     "md" => "markdown",
+    "cjs" => "typescript",
+    "mfs" => "typescript",
```

Comment:
typo? mfs -> mjs  
also doesnt include the typescript-equivalent "mts" and "cts" as mentioned in https://github.com/afnanenayet/diffsitter/issues/1217